### PR TITLE
Correct page dimmer, tab switch, progress label and popup heart icon positons

### PR DESCRIPTION
**Pull request by tarvos21**
Description:
- Page dimmer: add page dimmer HTML and correct selector
- Tab switch: comment out line "history: true"
- Progress bar and label: modify progress.slim and put kitchen-sink.js after default.js
- Popup 8 heart icons: make the segment square and define absolute positions for icons

Problem for discussion:
- h4 header like "Accordion, Table", the font-size is small, and color is black.

### CODE REVIEW

File: content/static/kitchen-sink/javascripts/kitchen-sink.js [hunk 1]
```diff
@@ -547,7 +547,7 @@ $(document)
     // tab
     $('.tab.demo .menu .item')
       .tab({
-        history: true,
+     // history: true,
```
Comment (xiaohanyu) [inline content/static/kitchen-sink/javascripts/kitchen-sink.js:550]: Why commenting this line out made it work? Don't quite understand. Could you explain something for me?

File: content/themes/semantic-ui/twitter/index.slim [hunk 2]
```diff
@@ -124,4 +124,3 @@
       == @items['/partials/examples/transition.slim'].compiled_content
 
 link href="/static/kitchen-sink/stylesheets/kitchen-sink.css" rel="stylesheet" type="text/css"
-script src="/static/kitchen-sink/javascripts/kitchen-sink.js"
```
Comment (xiaohanyu) [inline content/themes/semantic-ui/twitter/index.slim:127]: Still prefer put `kitchen-sink.js` to just themes pages. Later we may add some more pages, such as blog posts, templates, landing page, etc. And not all page need a `kitchen-sink.js` link.

**Review by xiaohanyu — COMMENTED** [review]
95% good, good job.

File: content/static/kitchen-sink/javascripts/kitchen-sink.js (hunk 1 above)
Comment (tarvos21) [reply content/static/kitchen-sink/javascripts/kitchen-sink.js:550]: I do not quite understand the "history" argument, either. In the documentation, it is called hash history: For static sites, or sites that only require simple history, in-page links can be used to trigger different local tab states. I do not understand what he is saying here. Maybe for simple static sites, history argument is not necessary.

File: content/themes/semantic-ui/twitter/index.slim (hunk 2 above)
Comment (tarvos21) [reply content/themes/semantic-ui/twitter/index.slim:127]: OK, I will try to move the progress part of JS code somewhere else and put kitchen-sink.js back.

**Review by xiaohanyu — APPROVED** [review]

**Pull request merged**